specifyweb/frontend/js_src/lib/components/TreeView/Tree.tsx

```diff
@@ -48,7 +49,7 @@ export function Tree<SCHEMA extends AnyTree>({
   searchBoxRef,
   baseUrl,
   setLastFocusedTree,
-  handleToggleEditingRanks,
+  handleToggleEditingRanks: onToggleEditingRanks,
```

Comment:
no 🤦
this is the opposite of how we do it)

do you remember how we have it as `onToggleEditingRanks: handleToggleEditingRanks` everywhere?
looks like you swapped it accidentally here